src/util/string_utils.cpp

```diff
@@ -37,16 +37,6 @@ std::string strip_string(const std::string &s)
   return s.substr(i, (j-i+1));
 }
 
-/// Given a string s, split into a sequence of substrings when separated by
```

Comment:
:pick: The commit message says `strip_string` but the function is called `split_string`